src/components/utils/ButtonBack/ButtonBack.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import './buttonBack.scss';
+
+export const BackButton = () => {
+  const navigate = useNavigate();
+
+  return (
+    <div className='back-button-container'>
+      <button
+        type="button"
+        className="back-button"
+        onClick={() => navigate(-1)}
```

Comment:
wyciągnij to do funkcji, np 
  const handleGoBack = () => {
    navigate(-1);
  };